mtags/src/main/scala/scala/meta/internal/pc/Completions.scala

```diff
@@ -1183,14 +1191,42 @@ trait Completions { this: MetalsGlobal =>
       override def contribute: List[Member] = {
         if (start < 0) Nil
         else {
-          typed.tpe.members.iterator
+
+          val overrideMembers: List[OverrideDefMember] =
+            typed.tpe.members.iterator
+              .filter(isOverridableMethod)
+              .map(OverrideCandidate.apply)
+              .filter { candidate =>
+                CompletionFuzzy.matchesSubCharacters(
+                  prefix,
+                  candidate.filterText
+                )
+              }
+              .map(_.toMember)
+              .toList
+
+          val allAbstractMembers = typed.tpe.members.iterator
             .filter(isOverridableMethod)
             .map(OverrideCandidate.apply)
-            .filter { candidate =>
-              CompletionFuzzy.matchesSubCharacters(prefix, candidate.filterText)
-            }
             .map(_.toMember)
+            .filter(_.sym.isAbstract)
             .toList
+            .map(_.edit.getNewText)
```

Comment:
I wonder if we could somehow reuse the rendering of the signatures? Alternatively, could we introduce some cache to avoid computing the type signatures + imports twice?

Reply:
This was the one point I didn't feel like addressed. I saw there were a couple of other places in the completions were cache was implemented but was sort of lost on how to implement it here.